<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Lab Software - My Docs</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-primary">
            <div class="container">
                <a class="navbar-brand" href="..">My Docs</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">Welcome to MkDocs</a>
                            </li>
                            <li class="navitem">
                                <a href="../Alarm-Information/" class="nav-link">Alarm Information</a>
                            </li>
                            <li class="navitem">
                                <a href="../COVID-Health-and-Safety-Plan/" class="nav-link">COVID Health and Safety Plan</a>
                            </li>
                            <li class="navitem">
                                <a href="../Chemical-Standard-Operating-Protocols-%28SOP%29/" class="nav-link">Chemical Standard Operating Protocols (SOP)</a>
                            </li>
                            <li class="navitem">
                                <a href="../Code-of-Conduct/" class="nav-link">Code of Conduct</a>
                            </li>
                            <li class="navitem">
                                <a href="../Computing-Hardware/" class="nav-link">Roberts Lab Computing</a>
                            </li>
                            <li class="navitem">
                                <a href="../DEI-Resources/" class="nav-link">DEI Resources</a>
                            </li>
                            <li class="navitem">
                                <a href="../Data-Management/" class="nav-link">Data Management</a>
                            </li>
                            <li class="navitem">
                                <a href="../Digital-Media/" class="nav-link">Digital Media</a>
                            </li>
                            <li class="navitem">
                                <a href="../Environment-and-Expectations/" class="nav-link">Environment and Expectations</a>
                            </li>
                            <li class="navitem">
                                <a href="../Experiment-Database/" class="nav-link">Experiment Database</a>
                            </li>
                            <li class="navitem">
                                <a href="../External-Communication-%26-Funding/" class="nav-link">External Communication & Funding</a>
                            </li>
                            <li class="navitem">
                                <a href="../Genomic-Resources/" class="nav-link">Genomic Resources</a>
                            </li>
                            <li class="navitem">
                                <a href="../Graduate-Research-Plan/" class="nav-link">Graduate Research Plan</a>
                            </li>
                            <li class="navitem">
                                <a href="../Home/" class="nav-link">Home</a>
                            </li>
                            <li class="navitem">
                                <a href="../Lab-Communication/" class="nav-link">Lab Communication</a>
                            </li>
                            <li class="navitem">
                                <a href="../Lab-Inventory/" class="nav-link">Lab Inventory</a>
                            </li>
                            <li class="navitem">
                                <a href="../Lab-Notebooks/" class="nav-link">Lab Notebooks</a>
                            </li>
                            <li class="navitem">
                                <a href="../Lab-Protocols/" class="nav-link">Lab Protocols</a>
                            </li>
                            <li class="navitem">
                                <a href="../Lab-Safety/" class="nav-link">Roberts Lab Safety</a>
                            </li>
                            <li class="navitem active">
                                <a href="./" class="nav-link">Lab Software</a>
                            </li>
                            <li class="navitem">
                                <a href="../Onboarding/" class="nav-link">Onboarding</a>
                            </li>
                            <li class="navitem">
                                <a href="../Pubathon/" class="nav-link">Pub-a-thon 2020</a>
                            </li>
                            <li class="navitem">
                                <a href="../Purchasing-%26-Reimbursement/" class="nav-link">SAFS Purchasing</a>
                            </li>
                            <li class="navitem">
                                <a href="../Undergraduate-contract/" class="nav-link">Undergraduate contract</a>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a rel="prev" href="../Lab-Safety/" class="nav-link">
                                    <i class="fa fa-arrow-left"></i> Previous
                                </a>
                            </li>
                            <li class="nav-item">
                                <a rel="next" href="../Onboarding/" class="nav-link">
                                    Next <i class="fa fa-arrow-right"></i>
                                </a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            
            
            
            
            
            
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<p>TOC <br />
<em> <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-hummingbird">hummingbird</a>  <br />
</em> <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software/#computer-roadrunner">roadrunner</a>
* <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-ostrich">ostrich</a>
* <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software/#computer-emu">emu</a>
* <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-woodpecker">woodpecker</a>
* <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-mox-hyak">Mox</a>
* <a href="https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-swan">swan</a></p>
<hr />
<h3 id="computer-hummingbird">Computer: hummingbird</h3>
<h4 id="os-os-x-1095">OS: OS X (10.9.5)</h4>
<p>FastQC (0.11.2)</p>
<pre><code> path = /usr/local/bioinformatics/FastQC/
</code></pre>
<p>Trimmomatic (0.30)</p>
<pre><code> path = /usr/local/bioinformatics/Trimmomatic-0.30/
</code></pre>
<p>Trinotate (Release 2014-07-08)</p>
<pre><code> path = /usr/local/bioinformatics/Trinotate_r20140708/
</code></pre>
<p>bedtools (2.22.1)</p>
<pre><code> path = /usr/local/bioinformatics/bedtools-2.22.1/
</code></pre>
<p>bismark (0.14.2)</p>
<pre><code> path = /usr/local/bioinformatics/bismark_v0.14.2/
</code></pre>
<p>bowtie (1.0.0)</p>
<pre><code> path = /usr/local/bioinformatics/bowtie-1.0.0/
</code></pre>
<p>bowtie2 (2.1.0)</p>
<pre><code> path = /usr/local/bioinformatics/bowtie2-2.1.0/
</code></pre>
<p>cufflinks (2.2.1.OSX_x86_64)</p>
<pre><code> path = /usr/local/bioinformatics/cufflinks-2.2.1.OSX_x86_64/
</code></pre>
<p>fastx_toolkit (0.0.13.2)</p>
<pre><code> path = /usr/local/bioinformatics/fastx_toolkit-0.0.13.2/
</code></pre>
<p>hmmer (3.1b1)</p>
<pre><code> path = /usr/local/bioinformatics/hmmer-3.1b1/
</code></pre>
<p>libgtextutils (0.6.1)</p>
<pre><code> path = /usr/local/bioinformatics/libgtextutils-0.6.1/
</code></pre>
<p>ncbi-blast (2.2.29+)</p>
<pre><code> path = /usr/local/bioinformatics/ncbi-blast-2.2.29+/
</code></pre>
<p>ngsplot (2.08)</p>
<pre><code> path = /usr/local/bioinformatics/ngsplot-2.08/
</code></pre>
<p>rsem (1.2.10)</p>
<pre><code> path = /usr/local/bioinformatics/rsem-1.2.10/
</code></pre>
<p>signalp (4.1)</p>
<pre><code> path = /usr/local/bioinformatics/signalp-4.1/
</code></pre>
<p>stacks (1.13)</p>
<pre><code> path = /usr/local/bioinformatics/stacks-1.13/
</code></pre>
<p>tophat (2.0.13.OSX_x86_64)</p>
<pre><code> path = /usr/local/bioinformatics/tophat-2.0.13.OSX_x86_64/
</code></pre>
<p>trinity (Release 2013-11-10)</p>
<pre><code> path = /usr/local/bioinformatics/trinityrnaseq_r20131110/
</code></pre>
<p>ipython (3.0.0)</p>
<p>Geneious (5.3.6)</p>
<hr />
<h3 id="computer-ostrich">Computer: ostrich</h3>
<h4 id="os-os-x-10116">OS: OS X (10.11.6)</h4>
<p>Anaconda3 (with R kernel and rpy2 for jupyter)</p>
<pre><code> path = /Users/Shared/anaconda3
</code></pre>
<p>FastQC (0.11.5)</p>
<pre><code> path = /Users/Shared/bioinformatics/FastQC/
</code></pre>
<p>Trimmomatic (0.30)</p>
<pre><code> path = /Users/Shared/bioinformatics/Trimmomatic-0.36/
</code></pre>
<p>Trinotate (3.0.1)</p>
<pre><code> path = /Users/Shared/bioinformatics/Trinotate-3.0.1/
</code></pre>
<p>bedtools (2.26.0)</p>
<pre><code> path = /Users/Shared/bioinformatics/bedtools2/
</code></pre>
<p>bismark (0.16.3)</p>
<pre><code> path = /Users/Shared/bioinformatics/bismark_v0.15.0/
</code></pre>
<p>bowtie (1.1.2)</p>
<pre><code> path = /Users/Shared/bioinformatics/bowtie1-1.1.2
</code></pre>
<p>bowtie2 (2.2.9)</p>
<pre><code> path = /Users/Shared/bioinformatics/bowtie2-2.2.9
</code></pre>
<p>celera assembler (8.3rc2)</p>
<pre><code>path = /Users/Shared/bioinformatics/wgs-8.3rc2
</code></pre>
<p>cufflinks (2.2.1.OSX_x86_64)</p>
<pre><code> path = /Users/Shared/bioinformatics/cufflinks-2.2.1.OSX_x86_64
</code></pre>
<p>hmmer (3.1b2)</p>
<pre><code> path = /Users/Shared/bioinformatics/hmmer-3.1b2-macosx-intel
</code></pre>
<p>ncbi-blast (2.5.0)</p>
<pre><code> path = /Users/Shared/bioinformatics/ncbi-blast-2.5.0+
</code></pre>
<p>ngsplot (2.08)</p>
<pre><code> path = /Users/Shared/bioinformatics/ngsplot
</code></pre>
<p>rsem (1.3.0)</p>
<pre><code> path = /Users/Shared/bioinformatics/RSEM-1.3.0
</code></pre>
<p>stacks (1.44)</p>
<pre><code> path = /Users/Shared/bioinformatics/stacks-1.44
</code></pre>
<p>tophat (2.1.1)</p>
<pre><code> path = /Users/Shared/bioinformatics/tophat-2.1.1.OSX_x86_64
</code></pre>
<p>trinity (2.3.2)</p>
<pre><code> path = /Users/Shared/bioinformatics/trinityrnaseq-Trinity-V2.3.2
</code></pre>
<hr />
<h3 id="computer-emu">Computer: emu</h3>
<h4 id="os-ubuntu-1604">OS: Ubuntu 16.04</h4>
<h4 id="cpus-16">CPU(s): 16</h4>
<h4 id="memory-48gb">Memory: 48GB</h4>
<h4 id="hdd-capacity-18tb">HDD Capacity: 1.8TB</h4>
<pre><code>$/home/shared
20200207_macau                BUSCO_v1.22                     include                         MultiQC                RepeatModeler-open-1.0.11   SparseAssembler
abacus                        comet                           isb-2.6.0+-changes-vers2.patch  ncbi-blast-2.6.0+-src  RepeatScout-1.0.5           sra-toolkit
bedtools-2.28.0               doc                             jellyfish-2.2.6                 nseg                   rmblast_2.6.0               src
BerkleyDB                     EMBOSS-6.6.0                    kallisto                        PBSuite_15.8.24        RNAMMER                     tmhmm-2.0c
bfg-1.13.0.jar                etc                             last-658                        pecan                  salmon-0.11.3-linux_x86_64  TransDecoder-3.0.1
bin                           fastqc_0.11.7                   lib                             pitchfork              salmon-1.2.1_linux_x86_64   TrimGalore-0.4.5
Bismark-0.19.1                gsl                             macau                           ProtDigSim             samtools-1.8                Trimmomatic-0.36
bissnp                        hmmer-2.3                       methpipe                        proteowiz              seqkit.v0.9.3               trinityrnaseq-Trinity-v2.4.0
BisSNP                        hmmer-3.1b2-linux-intel-x86_64  MethylExtract_1.9.1             pyfaidx-0.5.5.2        SGE6.2u5p2                  trinityrnaseq-Trinity-v2.8.4
blastdb                       IGV_Linux_2.6.1                 miniconda2                      ReAdW                  share                       Trinotate-3.0.2
bowtie2-2.3.4.1-linux-x86_64  igvtools                        miniconda3                      RECON-1.08             Sibelia-3.0.7-Linux         var
bsmap                         igv-webapp                      MSFileReader                    RepeatMasker-4.0.7     signalp-4.1
</code></pre>
<pre><code>/usr/bin
R (Microsoft Open R 3.5.1)
</code></pre>
<pre><code>/home/shared/miniconda3/bin
jupyter (with R Kernel)
</code></pre>
<hr />
<h3 id="computer-roadrunner">Computer: roadrunner</h3>
<h4 id="os-ubuntu-1604_1">OS: Ubuntu 16.04</h4>
<h4 id="cpus-16_1">CPU(s): 16</h4>
<h4 id="memory-48gb_1">Memory: 48GB</h4>
<h4 id="hdd-capacity-18tb_1">HDD Capacity: 1.8TB</h4>
<pre><code>$/home/shared
alpaca           Bismark-v0.17.0               bwa-0.7.17      igv-webapp              macau                  ncbi-blast-2.7.1+-src  RepeatMasker-4.0.7         SOAPdenovo2-r241
anaconda3        BisSNP                        ectools-0.1     jellyfish-2.2.6         meraculous-2.2.5       parallel-20180422      rmblast_2.6.0              TrimGalore-0.4.5
bbmap_38.44      bowtie2-2.3.3.1-linux-x86_64  EMBOSS-6.6.0    kallisto_linux-v0.43.1  Meraculous-v2.2.5      PBSuite_15.8.24        salmon-1.2.1_linux_x86_64  wgs-assembler
bedtools-2.28.0  bowtie2-2.3.4.1-linux-x86_64  fastqc_v0.11.7  kmergenie-1.7044        MultiQC                pecan                  samtools-1.6
bin              BSMap                         IGVTools        Log-Log4perl-1.49       ncbi-blast-2.6.0+-src  pyfaidx-0.5.3.1        samtools-1.8
</code></pre>
<pre><code>/usr/bin
cutadapt
R (Microsoft Open R 3.5.1)
</code></pre>
<pre><code>/home/shared/anaconda3/bin
jupyter (with R Kernel)
</code></pre>
<h5 id="jbrowse-genome-viewer-v1154"><a href="https://jbrowse.org/">JBrowse Genome Viewer v1.15.4</a></h5>
<p>http://roadrunner.fish.washington.edu/jbrowse/</p>
<hr />
<h3 id="computer-mox-hyak">Computer: Mox (Hyak)</h3>
<h4 id="os-red-hat-enterprise-72-login-nodes">OS: Red Hat Enterprise 7.2 (<a href="https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#login-node">login nodes</a>)</h4>
<h4 id="os-centos-linux-721511-build-interactive-nodes-execute-nodes">OS: CentOS Linux 7.2.1511 (<a href="https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#build-node">build</a>,  <a href="https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#interactive-node">interactive nodes</a>, &amp; <a href="https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#execute-node">execute nodes</a>)</h4>
<pre><code>$/gscratch/srlab/programs
2bRAD_GATK            busco-v3      git-sym         mdust                    perl                samtools-1.4              supernova-2.0.0
argparse-1.4.0            bwa-0.7.15    graphviz-2.40.1     megahit_v1.1.4_LINUX_CPUONLY_x86_64-bin  picard_2.18.4           samtools-1.9              tmhmm-2.0c
Augustus-3.3.2            bwa-0.7.17    hisat2-2.1.0        MetaGeneMark_linux_64_3.38       picard-2.9.1            scripts                   transdecoder
bamtools-2.5.1            canu      hmmer-2.3       Metassembler                 pigz-2.4            seqtk-1.3                 TransDecoder-v5.3.0
bbmap_38.34           celera        hmmer-3.1b2     minimap                  pilon               setuptools-36.0.1             TransDecoder-v5.5.0
bcftools-1.9              cmake-3.12.1  hmmer-3.2.1     mummer                   pitchfork           signalp-4.1               TrimGalore-0.4.5
bcl2fastq-v2.20           cutadapt-1.16 htslib-1.9      MUMmer3.23               platanus_1.2.4          smrtanalysis_2.3.0.140936.run     Trimmomatic-0.36
bedtools-2.27.1           decorator-4.0.11  interproscan-5.31-70.0  ncbi-blast-2.6.0+            pyfaidx-0.5.5.2         snap                  Trinity-v2.8.3
bin               FastaIndex    jellyfish-2.2.10    ncbi-blast-2.8.1+            pyScaf              SOAPdenovo2               Trinotate-v3.1.1
Bismark-0.19.0            FastQC-0.11.5 jsoncpp-1.8.4       ncbi-blast-2.8.1+_orginal        quast-4.5           SOAPec_bin_v2.03              wtdbg-2.1_x64-linux
Bismark-0.21.0            gapcloser-1.12    kmergenie-1.7048    networkx-1.11                racon               SPAdes-3.13.0-Linux
Bismark-0.21.0_dev        GARM_v0.7.5   last-852        networkx2                redundans           SparseAssembler
bowtie2-2.1.0             GATK      likelybin-0.1.0     parallel-20180822            resources           sqlite
bowtie2-2.3.4.1-linux-x86_64  gcc       maker-2.31.10       PBSuite                  RNAMMER-1.2             SSPACE-LongRead_v1-1
bsmap-2.89            get-pip.py    MaSuRCA-3.2.3       pecan                    salmon-0.11.2-linux_x86_64  SSPACE-STANDARD-3.0_linux-x86_64

</code></pre>
<hr />
<h3 id="computer-woodpecker">Computer: woodpecker</h3>
<h4 id="os-windows-7-enterprise-64-bit">OS: Windows 7 Enterprise (64-bit)</h4>
<p>Microsoft Office 2016 Pro Plus</p>
<pre><code>      path = C:\Program Files\Microsoft Office
</code></pre>
<p>Protein Digest Simulator v2.2.6138.19320</p>
<pre><code>      path = C:\Program Files (x86)\ProteinDigestionSimulator
</code></pre>
<p>Proteowizard v3.0.10577</p>
<pre><code>      path = C:\Program Files\ProteoWizard\ProteoWizard 3.0.10577
</code></pre>
<p>Skyline v3.6</p>
<pre><code>      path =
</code></pre>
<hr />
<h3 id="computer-swan">Computer: swan</h3>
<h4 id="os-windows-7-enterprise-64-bit_1">OS: Windows 7 Enterprise (64-bit)</h4>
<p>Microsoft Office Professional 2016 64-bit
- Access
- Excel
- Power Point
- Word</p>
<p>R v3.4.2</p>
<p>R Studio 1.1383</p>
<p>Skyline v3.7.0.11317</p>
<p>Skyline Daily v3.7.1.11446</p>
<p>Virtual Clonedrive v5.5.0.0</p>
<p>ProteoWizard 3.0.11516 64-bit
- MSConvert
- SeeMS</p>
<p>LabX v. 8.0.0 Build 3444</p>
<p>Protein Digest Simulator v2.2.6471; September 19, 2017</p></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>
                    From here you can search these documents. Enter
                    your search terms below.
                </p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
