
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.1.2, mkdocs-material-6.2.5">
    
    
      
        <title>Genomic Resources - Roberts Lab Handbook</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.15aa0b43.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.75751829.min.css">
        
      
    
    
    
      
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto",-apple-system,BlinkMacSystemFont,Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono",SFMono-Regular,Consolas,Menlo,monospace}</style>
      
    
    
    
    
      
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="slate" data-md-color-primary="" data-md-color-accent="">
      
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#related-resources" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid" aria-label="Header">
    <a href=".." title="Roberts Lab Handbook" class="md-header-nav__button md-logo" aria-label="Roberts Lab Handbook">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header-nav__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header-nav__title" data-md-component="header-title">
      <div class="md-header-nav__ellipsis">
        <div class="md-header-nav__topic">
          <span class="md-ellipsis">
            Roberts Lab Handbook
          </span>
        </div>
        <div class="md-header-nav__topic">
          <span class="md-ellipsis">
            
              Genomic Resources
            
          </span>
        </div>
      </div>
    </div>
    
      <label class="md-header-nav__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" data-md-state="active" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <button type="reset" class="md-search__icon md-icon" aria-label="Clear" data-md-component="search-reset" tabindex="-1">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    




<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Roberts Lab Handbook" class="md-nav__button md-logo" aria-label="Roberts Lab Handbook">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    Roberts Lab Handbook
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Alarm-Information/" class="md-nav__link">
        Alarm Information
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Chemical-Standard-Operating-Protocols-%28SOP%29/" class="md-nav__link">
        Chemical Standard Operating Protocols (SOP)
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Code-of-Conduct/" class="md-nav__link">
        Code of Conduct
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Computing-Hardware/" class="md-nav__link">
        Roberts Lab Computing
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../COVID-Health-and-Safety-Plan/" class="md-nav__link">
        COVID Health and Safety Plan
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Data-Management/" class="md-nav__link">
        Data Management
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../DEI-Resources/" class="md-nav__link">
        DEI Resources
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Digital-Media/" class="md-nav__link">
        Digital Media
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Environment-and-Expectations/" class="md-nav__link">
        Environment and Expectations
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Experiment-Database/" class="md-nav__link">
        Experiment Database
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../External-Communication-%26-Funding/" class="md-nav__link">
        External Communication & Funding
      </a>
    </li>
  

    
      
      
      


  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Genomic Resources
      </a>
      
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Graduate-Research-Plan/" class="md-nav__link">
        Graduate Research Plan
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Home/" class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Welcome to MkDocs
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Communication/" class="md-nav__link">
        Lab Communication
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Inventory/" class="md-nav__link">
        Lab Inventory
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Notebooks/" class="md-nav__link">
        Lab Notebooks
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Protocols/" class="md-nav__link">
        Lab Protocols
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Safety/" class="md-nav__link">
        Roberts Lab Safety
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Lab-Software/" class="md-nav__link">
        Lab Software
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Onboarding/" class="md-nav__link">
        Onboarding
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Pubathon/" class="md-nav__link">
        Pub-a-thon 2020
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Purchasing-%26-Reimbursement/" class="md-nav__link">
        SAFS Purchasing
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../README.md" class="md-nav__link">
        None
      </a>
    </li>
  

    
      
      
      


  
  
  
    <li class="md-nav__item">
      <a href="../Undergraduate-contract/" class="md-nav__link">
        Undergraduate contract
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                
                <p>Here we try to compile genomic resources such that they are readily available and somewhat described. An effort will be made to keep respective index files alongside so these files can be directly used in IGV <em>etc</em>.</p>
<h5 id="related-resources">Related Resources</h5>
<ul>
<li>
<p>Archived Versions of this page - <a href="http://htmlpreview.github.io/?https://github.com/RobertsLab/resources/blob/master/archive/Genomic-Resources.091419.html">091319</a>;  </p>
</li>
<li>
<p><a href="https://docs.google.com/spreadsheets/d/1_XqIOPVHSBVGscnjzDSWUeRL7HUHXfaHxVzec-I-8Xk/edit?usp=sharing">Nightingales (Google Sheet)</a> : Database of all raw high-throughput sequencing data</p>
</li>
</ul>
<h1 id="chionoecetes-bairdi"><em>Chionoecetes bairdi</em></h1>
<h2 id="genomes">Genomes</h2>
<ul>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_genome_v1.01.fasta">cbai_genome_v1.01.fasta</a> (18MB)</p>
</li>
<li>
<p>MD5 = <code>5a08d8b0651484e3ff75fcf032804596</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/09/24/Assembly-Assessment-BUSCO-C.bairdi-Genome-v1.01-on-Mox.html">BUSCOs</a>:
  <code>C:0.4%[S:0.3%,D:0.1%],F:0.2%,M:99.4%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_genome_v1.01.fasta.fai">cbai_genome_v1.01.fasta.fai</a></li>
</ul>
</li>
<li>
<p>Assembly from <a href="https://robertslab.github.io/sams-notebook/2020/09/23/Data-Wrangling-Subsetting-cbai_genome_v1.0-Assembly-with-faidx.html">20200923</a></p>
<ul>
<li>
<p>Q7-filtered NanoPore data. Includes <em>Hematodinium</em>-infected sample.</p>
</li>
<li>
<p>Subset of <code>cbai_genome_v1.0.fasta</code> with contigs &gt;1000bp</p>
</li>
</ul>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_genome_v1.0.fasta">cbai_genome_v1.0.fasta</a> (19MB)</p>
</li>
<li>
<p>MD5 = <code>2f3b651bb0b875b0287e71e315cad59a</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/09/18/Assembly-Assessment-BUSCO-C.bairdi-Genome-v1.0-on-Mox.html">BUSCOs</a>:
  <code>C:0.4%[S:0.3%,D:0.1%],F:0.3%,M:99.3%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_genome_v1.0.fasta.fai">cbai_genome_v1.0.fasta.fai</a></li>
</ul>
</li>
<li>
<p>Assembly from <a href="https://robertslab.github.io/sams-notebook/2020/09/17/Genome-Assembly-C.bairdi-cbai_v1.0-Using-All-NanoPore-Data-With-Flye-on-Mox.html">20200917</a></p>
<ul>
<li>Q7-filtered NanoPore data. Includes <em>Hematodinium</em>-infected sample.</li>
</ul>
</li>
</ul>
<h2 id="transcriptomes">Transcriptomes</h2>
<ul>
<li>
<p><a href="https://docs.google.com/spreadsheets/d/1d17yg5F5gKKC66O8QkTIlPxljJeuX7ZsG46pkBr1lNQ/edit#gid=0">RNA-seq sample list</a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v3.1.fasta">cbai_transcriptome_v3.1.fasta</a></p>
</li>
<li>
<p>MD5 = <code>aeec8ffbf8fa44fb1750caee6abaf68a</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/06/05/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-Transcriptome-v3.1.html">BUSCOs</a>: <code>C:96.5%[S:40.3%,D:56.2%],F:2.2%,M:1.3%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v3.1.fasta.fai">cbai_transcriptome_v3.1.fasta.fai</a></li>
</ul>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200608_cbai_diamond_blastx_v2.1_v3.1/cbai_transcriptome_v3.1.blastx.outfmt6">BLASTx annotation</a> (outfmt6)</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200828_cbai_trinotate_transcriptome-v3.1/20200828.cbai_transcriptome_v3.1.fasta.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-UW with <em>non_Alveolata</em>. <a href="https://robertslab.github.io/sams-notebook/2020/06/05/Sequence-Extractions-C.bairdi-Transcriptomes-v2.0-and-v3.0-Excluding-Alveolata-with-MEGAN6-on-Swoose.html">Derived from <code>cbai_transcriptome_v3.0.fasta</code></a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v3.0.fasta">cbai_transcriptome_v3.0.fasta</a></p>
</li>
<li>
<p>Assembly from <a href="https://robertslab.github.io/sams-notebook/2020/05/18/Transcriptome-Assembly-C.bairdi-All-Pooled-RNAseq-Data-Without-Taxonomic-Filters-with-Trinity-on-Mox.html">20200518</a></p>
</li>
<li>
<p>MD5 = <code>5516789cbad5fa9009c3566003557875</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/05/19/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-Transcriptome-v3.0.html">BUSCOs</a>: <code>C:97.6%[S:39.1%,D:58.5%],F:1.6%,M:0.8%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v3.0.fasta.fai">cbai_transcriptome_v3.0.fasta.fai</a></li>
</ul>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200519_cbai_diamond_blastx_transcriptome_v3.0/20200518.C_bairdi.Trinity.blastx.outfmt6">BLASTx annotation</a> (outfmt6)</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200526_cbai_trinotate_transcriptome-v3.0/20200526.cbai_transcriptome_v3.0.fasta.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-UW with <em>no taxonomic filter</em>.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v2.1.fasta">cbai_transcriptome_v2.1.fasta</a></p>
</li>
<li>
<p>MD5 = <code>1fb788175f9bb7cd5145370a399ae857</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/06/05/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-Transcriptome-v2.1.html">BUSCOs</a>: <code>C:98.3%[S:25.2%,D:73.1%],F:1.4%,M:0.3%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v2.1.fasta.fai">cbai_transcriptome_v2.1.fasta.fai</a></li>
</ul>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200608_cbai_diamond_blastx_v2.1_v3.1/cbai_transcriptome_v2.1.blastx.outfmt6">BLASTx annotation</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200831_cbai_trinotate_transcriptome-v2.1/20200831.cbai_transcriptome_v2.1.fasta.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-GW, 2020-UW with <em>non_Alveolata</em>. <a href="https://robertslab.github.io/sams-notebook/2020/06/05/Sequence-Extractions-C.bairdi-Transcriptomes-v2.0-and-v3.0-Excluding-Alveolata-with-MEGAN6-on-Swoose.html">Derived from <code>cbai_transcriptome_v2.0.fasta</code></a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v2.0.fasta">cbai_transcriptome_v2.0.fasta</a></p>
</li>
<li>
<p>Also referred to as <a href="https://robertslab.github.io/sams-notebook/2020/05/02/Transcriptome-Assembly-C.bairdi-All-RNAseq-Data-Without-Taxonomic-Filters-with-Trinity-on-Mox.html"><code>20200507.C_bairdi.Trinity.fasta</code></a>.</p>
</li>
<li>
<p>MD5 = <code>01adbd54298495c147767b19ee5c0de9</code>
https://gannet.fish.washington.edu/Atumefaciens/20200526_cbai_trinotate_transcriptome-v3.0/20200526.cbai.trinotate.go_annotations.txt</p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/05/08/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-v2.0-Transcriptome.html">BUSCOs</a>: <code>C:98.8%[S:24.9%,D:73.9%],F:0.9%,M:0.3%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v2.0.fasta.fai">cbai_transcriptome_v2.0.fasta.fai</a></li>
</ul>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200508_cbai_diamond_blastx_transcriptome-v2.0/20200507.C_bairdi.Trinity.blastx.outfmt6">BLASTx annotation</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200513_cbai_trinotate_transcriptome-v2.0/20200513.cbai.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-GW, 2020-UW with <em>no taxonomic filter</em>.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.7.fasta">cbai_transcriptome_v1.7.fasta</a></p>
</li>
<li>
<p>MD5 = <code>032d1f81c7744736ebeefe7f63ed6d95</code></p>
</li>
<li>
<p>Assembly from <a href="https://robertslab.github.io/sams-notebook/2020/05/27/Transcriptome-Assembly-C.bairdi-All-Pooled-Arthropoda-only-RNAseq-Data-with-Trinity-on-Mox.html">20200527</a></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.7.fasta.fai">cbai_transcriptome_v1.7.fasta.fai</a> :
    <code>https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.7.fasta.fai</code></li>
</ul>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/05/27/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-Transcriptome-v1.7.html">BUSCOs</a>: <code>C:86.7%[S:66.5%,D:20.2%],F:8.2%,M:5.1%,n:978</code></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200527_cbai_diamond_blastx_transcriptome_v1.7/cbai_transcriptome_v1.7.blastx.outfmt6">BLASTx Annotation</a> (outfmt6)</p>
</li>
<li>
<p>[GO Terms Annotation] (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-UW with <em>Arthropoda</em> only reads.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.6.fasta">cbai_transcriptome_v1.6.fasta</a></p>
</li>
<li>
<p>MD5 = <code>46d77ce86cdbbcac26bf1a6cb820651e</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.6.fasta.fai">cbai_transcriptome_v1.6.fasta.fai</a> :
  <code>https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.6.fasta.fai</code></li>
</ul>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/05/19/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-Transcriptome-v1.6.html">BUSCOs</a>: <code>C:91.7%[S:62.6%,D:29.1%],F:6.2%,M:2.1%,n:978</code></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200519_cbai_diamond_blastx_transcriptome_v1.6/cbai_transcriptome_v1.6.blastx.outfmt6">BLASTx Annotation</a> (outfmt6)</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200520_cbai_trinotate_transcriptome-v1.6/20200520.cbai_transcriptome_v1.6.fasta.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-GW, 2020-UW with <em>Arthropoda</em> only reads.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.5.fasta">cbai_transcriptome_v1.5.fasta</a></p>
</li>
<li>
<p>MD5 = <code>e61d68c45728ffbb91e3d34c087d9aa9</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/04/07/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-MEGAN-Transcriptome.html">BUSCOs</a>: C:91.8%[S:64.0%,D:27.8%],F:5.9%,M:2.3%,n:978</p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.5.fasta.fai">cbai_transcriptome_v1.5.fasta.fai</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.5.fasta.fai</code></li>
</ul>
</li>
<li>
<p>Updated assembly from <a href="https://robertslab.github.io/sams-notebook/2020/03/30/Transcriptome-Assembly-C.bairdi-with-MEGAN6-Taxonomy-specific-Reads-with-Trinity-on-Mox.html">20200330</a>. Also referred to as <code>20200408.C_bairdi.megan.Trinity.fasta</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/04/08/Transcriptome-Annotation-C.bairdi-MEGAN-Trinity-Assembly-Using-DIAMOND-BLASTx-on-Mox.html">BLASTx Annotation</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200409_cbai_trinotate_megan/20200409.cbai.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-GW with Arthropoda only reads.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.0.fasta">cbai_transcriptome_v1.0.fasta</a></p>
</li>
<li>
<p>MD5 = <code>fb28a203154b44b67ec2e2476d96d326</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/02/07/Transcriptome-Assessment-BUSCO-Metazoa-on-C.bairdi-MEGAN-Transcriptome.html">BUSCOs</a>: <code>C:85.5%[S:64.7%,D:20.8%],F:9.3%,M:5.2%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.0.fasta.fai">cbai_transcriptome_v1.0.fasta.fai</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/cbai_transcriptome_v1.0.fasta.fasta.fai</code></li>
</ul>
</li>
<li>
<p>Initial Trinity assembly from <a href="https://robertslab.github.io/sams-notebook/2020/01/22/Transcriptome-Assembly-C.bairdi-with-MEGAN6-Taxonomy-specific-Reads-with-Trinity-on-Mox.html">20200122</a></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/01/23/Transcriptome-Annotation-C.bairdi-MEGAN-Trinity-Assembly-Using-DIAMOND-BLASTx-on-Mox.html">BLASTx Annotation</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200126_cbai_trinotate_megan/20200126.cbai.trinotate.go_annotations.txt">GO Terms Annotation</a> (Trinotate)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019 with <em>Arthropoda</em> only reads.</p>
</li>
</ul>
<hr />
<h1 id="crassostrea-gigas"><em>Crassostrea gigas</em></h1>
<h5 id="related-resources_1">Related Resources</h5>
<ul>
<li>
<p><a href="https://figshare.com/articles/Compilation_of_DNA_Methylation_Genome_Feature_Tracks_Crassostrea_gigas_/1456226">Compilation of DNA Methylation Genome Feature Tracks (<em>Crassostrea gigas</em>)</a> <em>circa 2015</em></p>
</li>
<li>
<p><a href="http://onsnetwork.org/halfshell/2015/02/27/re-defining-cgigas-canonical-features/">Re-defining Cgigas Canonical features</a> <em>circa 2015</em></p>
</li>
<li>
<p><a href="http://gigaton.sigenae.org/ngspipelines/#!/NGSpipelines/Crassostrea%20gigas%20-%20GIGATON">Gigaton</a></p>
</li>
<li>
<p><a href="https://github.com/RobertsLab/resources/blob/master/html/tjgr.markdown">TJGR</a></p>
</li>
</ul>
<h2 id="genome">Genome:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel.fa">Crassostrea_gigas.oyster_v9.dna_sm.toplevel.fa</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel.fa</code></p>
</li>
<li>
<p>MD5 = 6de9d1239eb10ea0545bed6c4e746d6c</p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>) :
    <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel.fa.fai</code></p>
</li>
</ul>
<h2 id="bisulfite-genome">Bisulfite Genome:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel_bisulfite.tar.gz">Crassostrea_gigas.oyster_v9.dna_sm.toplevel_bisulfite.tar.gz</a> :
  <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel_bisulfite.tar.gz</code></p>
<ul>
<li>Gzipped tarball of bisulfite genome for use with Bismark</li>
<li>Creation details <a href="https://robertslab.github.io/sams-notebook/2019/02/21/Data-Management-Create-C.gigas-Bisulfite-Genome-with-Bismark-on-Mox.html">here</a></li>
</ul>
</li>
</ul>
<h2 id="genome-feature-tracks">Genome Feature Tracks</h2>
<ul>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_gene.gff">Cgigas_v9_gene.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_gene.gff</code></p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_exon.gff">Cgigas_v9_exon.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_exon.gff</code></p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_intron.gff">Cgigas_v9_intron.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_intron.gff</code></p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_TE.gff">Cgigas_v9_TE.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_TE.gff</code></p>
</li>
<li>
<p>Contains <a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_TE-TANDEMREPEAT.gff">Tandem Repeats</a> and <a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_TE-WUBLASTX.gff">wublastx</a> features.</p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_CG.gff">Cgigas_v9_CG.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_CG.gff</code></p>
</li>
<li>
<p>index: <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_CG.gff.idx</code></p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_1k5p_gene_promoter.gff">Cgigas_v9_1k5p_gene_promoter.gff</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_1k5p_gene_promoter.gff</code></p>
</li>
<li>
<p><a href="https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_COMP_gene_prom_TE.bed">Cgigas_v9_COMP_gene_prom_TE.bed</a> : <code>https://eagle.fish.washington.edu/trilobite/Crassostrea_gigas_v9_tracks/Cgigas_v9_COMP_gene_prom_TE.bed</code></p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.40.gff3">Crassostrea_gigas.oyster_v9.40.gff3</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.40.gff3</code></p>
</li>
<li>
<p>MD5 = 90a747fbc94a0a9225c43f75cc40b9db</p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.40.abinitio.gff3">Crassostrea_gigas.oyster_v9.40.abinitio.gff3</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.40.abinitio.gff3</code></p>
</li>
<li>
<p>MD5 = c2a8c388f5a8afb22a115d61dee3dda0</p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.40_mRNA.gff3">Crassostrea_gigas.oyster_v9.40_mRNA.gff3</a></p>
</li>
<li><code>grep "mRNA" Crassostrea_gigas.oyster_v9.40.gff3 &gt; Crassostrea_gigas.oyster_v9.40_mRNA.gff3</code></li>
</ul>
<hr />
<h1 id="crassostrea-virginica"><em>Crassostrea virginica</em></h1>
<p><a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/002/022/765/GCF_002022765.2_C_virginica-3.0/">NCBI FTP</a></p>
<h2 id="genomes_1">Genomes:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300.fa">Cvirginica_v300.fa</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300.fa</code></p>
<ul>
<li>
<p>MD5 = f9135e323583dc77fc726e9df2677a32</p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300.fa.fai">Cvirginica_v300.fa.fai</a> :
  <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300.fa.fai</code></p>
</li>
</ul>
</li>
<li>
<p><a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/002/022/765/GCF_002022765.2_C_virginica-3.0/GCF_002022765.2_C_virginica-3.0_genomic.fna.gz">GCF_002022765.2_C_virginica-3.0_genomic.fna</a> : <code>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/002/022/765/GCF_002022765.2_C_virginica-3.0/GCF_002022765.2_C_virginica-3.0_genomic.fna.gz</code></p>
<ul>
<li>compressed version of <code>Cvirginica_v300.fa</code> (same files)</li>
</ul>
</li>
</ul>
<h2 id="bisulfite-genomes">Bisulfite Genomes:</h2>
<ul>
<li><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300_bisulfite.tar.gz">Cvirginica_v300_bisulfite.tar.gz</a> :
<code>http://owl.fish.washington.edu/halfshell/genomic-databank/Cvirginica_v300_bisulfite.tar.gz</code></li>
<li>Gzipped tarball of bisulfite genome for use with Bismark</li>
<li>Creation details <a href="https://robertslab.github.io/sams-notebook/2019/02/21/Data-Management-Create-C.virginica-Bisulfite-Genome-wit-Bismark-on-Mox.html">here</a></li>
</ul>
<h2 id="genome-feature-tracks_1">Genome Feature Tracks</h2>
<ul>
<li>
<p><a href="http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_Gnomon_mRNA.gff3">C_virginica-3.0_Gnomon_mRNA.gff3</a> : <code>http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_Gnomon_mRNA.gff3</code></p>
</li>
<li>
<p><a href="http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_Gnomon_exon.bed">C_virginica-3.0_Gnomon_exon.bed</a> : <code>http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_Gnomon_exon.bed</code></p>
</li>
<li>
<p><a href="http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_intron.bed">C_virginica-3.0_intron.bed</a> : <code>http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_intron.bed</code></p>
</li>
<li>
<p><a href="http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_CG-motif.bed">C_virginica-3.0_CG-motif.bed</a> : <code>http://eagle.fish.washington.edu/Cvirg_tracks/C_virginica-3.0_CG-motif.bed</code></p>
</li>
<li>
<p>MD5 = f88c171bccf45a6f3afcf455b6be810f</p>
</li>
<li>
<p>Dead link in this Jupyter Notebook obscures details on how this was generated (via Galaxy):</p>
<ul>
<li>https://github.com/sr320/nb-2018/blob/master/C_virginica/22-CG-track.ipynb</li>
</ul>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/C_virginica-3.0_TE-all.gff">C_virginica-3.0_TE-all.gff</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/C_virginica-3.0_TE-all.gff</code>      </p>
</li>
<li>
<p>MD5 = d0d81fc6cf7525bc2c61984bee23521b    </p>
</li>
<li>
<p><a href="http://onsnetwork.org/kubu4/2018/08/28/transposable-element-mapping-crassostrea-virginica-genome-cvirginica_v300-using-repeatmasker-4-07/">Details</a></p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/C_virginica-3.0_TE-Cg.gff">C_virginica-3.0_TE-Cg.gff</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/C_virginica-3.0_TE-Cg.gff</code>      </p>
</li>
<li>
<p>MD5 = 83cd753c171076464fee1165b7e1c6ba</p>
</li>
<li>
<p><a href="http://onsnetwork.org/kubu4/2018/08/28/transposable-element-mapping-crassostrea-virginica-genome-cvirginica_v300-using-repeatmasker-4-07/">Details</a></p>
</li>
</ul>
<hr />
<h1 id="hematodinium-sp-host-chionoecetes-bairdi"><em>Hematodinium sp.</em> (Host: <em>Chionoecetes bairdi</em>)</h1>
<h2 id="transcriptomes_1">Transcriptomes</h2>
<ul>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta">hemat_transcriptome_v1.5.fasta</a></p>
</li>
<li>
<p>MD5 = <code>b8d4a3c1bad2e07da8431bf70bdabfdd</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/03/31/Transcriptome-Assessment-BUSCO-Metazoa-on-Hematodinium-MEGAN-Transcriptome.html">BUSCOs</a>: <code>C:25.6%[S:20.7%,D:4.9%],F:11.7%,M:62.7%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta.fai">hemat_transcriptome_v1.5.fasta.fai</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.5.fasta.fai</code></li>
</ul>
</li>
<li>
<p>Updated assembly from <a href="https://robertslab.github.io/sams-notebook/2020/03/30/Transcriptome-Assembly-Hematodinium-with-MEGAN6-Taxonomy-specific-Reads-with-Trinity-on-Mox.html">20200330</a>.</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200331_hemat_diamond_blastx_megan/20200408.hemat.megan.Trinity.fasta.blastx.outfmt6">BLASTx Annotation</a> (txt; 355KB)</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200408_hemat_trinotate_megan/20200408.hemat.trinotate.go_annotations.txt">Trinotate GO Terms Annotation</a> (txt; 2.3MB)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019, 2020-GW with <em>Alveolata</em> only reads.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.0.fasta">hemat_transcriptome_v1.0.fasta</a> (3.9MB)</p>
</li>
<li>
<p>MD5 = <code>fa5eb74767d180af5265d2d1f80b6430</code></p>
</li>
<li>
<p><a href="https://robertslab.github.io/sams-notebook/2020/02/07/Transcriptome-Assessment-BUSCO-Metazoa-on-Hematodinium-MEGAN-Transcriptome.html">BUSCOs</a>: <code>C:25.1%[S:19.2%,D:5.9%],F:9.5%,M:65.4%,n:978</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.0.fasta.fai">hemat_transcriptome_v1.0.fasta.fai</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/hemat_transcriptome_v1.0.fasta.fai</code></li>
</ul>
</li>
<li>
<p>Initial Trinity assembly from <a href="https://robertslab.github.io/sams-notebook/2020/01/22/Transcriptome-Assembly-Hematodinium-with-MEGAN6-Taxonomy-specific-Reads-with-Trinity-on-Mox.html">20200122</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200123_hemat_diamond_blastx_megan/20200122.hemat.megan.Trinity.blastx.outfmt6">BLASTx Annotation</a> (txt; 308KB)</p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/Atumefaciens/20200126_hemat_trinotate_megan/20200126.hemat.trinotate.go_annotations.txt">Trinotate GO Terms Annotation</a> (txt; 2.1MB)</p>
</li>
<li>
<p>internal short-hand: includes 2018, 2019 with <em>Alveolata</em> only reads.</p>
</li>
</ul>
<hr />
<h1 id="metacarcinus-magister-cancer-magister"><em>Metacarcinus magister</em> (<em>Cancer magister</em>)</h1>
<h2 id="genome_1">Genome:</h2>
<ul>
<li>
<p><code>mmag_pilon_scaffolds.fasta</code></p>
</li>
<li>
<p>MD5 = 5dfa2ba11edf0ff8191f112e0b1378d1</p>
</li>
<li>
<p>Not shared publicly until permission received from NOAA.</p>
</li>
<li>
<p>Roberts Lab members can access on Owl: <code>/web/halfshell/genomic-databank/mmag_pilon_scaffolds.fasta</code></p>
</li>
<li>
<p>Original filename: <code>pilon_scaffolds.fasta</code></p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><code>mmag_pilon_scaffolds.fasta.fai</code></li>
</ul>
</li>
</ul>
<hr />
<h1 id="ostrea-lurida"><em>Ostrea lurida</em></h1>
<h2 id="genome_2">Genome:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081.fa">Olurida_v081.fa</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081.fa</code></p>
</li>
<li>
<p>MD5 = 3ac56372bd62038f264d27eef0883bd3</p>
</li>
<li>
<p>This is <code>v080</code> with only contigs &gt; 1000bp. Details of how <code>v080</code> was reduced found <a href="https://github.com/sr320/nb-2018/blob/master/O_lurida/04-Pbjelly-10k-mapping.ipynb">here</a>.</p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081.fa.fai">Olurida_v081.fa.fai</a> :
<code>http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081.fa.fai</code></li>
</ul>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v080.fa">Olurida_v080.fa</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v080.fa</code></p>
</li>
<li>
<p>MD5 = 9258398f554493e08fdc30e9c1409864</p>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>)</p>
<ul>
<li><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v080.fa.fai">Olurida_v080.fa.fai</a> :
<code>http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v080.fa.fai</code></li>
</ul>
</li>
<li>
<p>Also known as <code>pbjelly_sjw_01</code>. Details found <a href="https://github.com/sr320/LabDocs/blob/master/jupyter_nbs/sam/20171130_emu_pbjelly.ipynb">here</a>, though confirmation would be good.</p>
</li>
</ul>
<p>## Bisulfite Genomes:</p>
<pre><code>- [Olurida_v080_bisulfite.tar.gz](http://owl.fish.washington.edu/halfshell/genomic-databank/Crassostrea_gigas.oyster_v9.dna_sm.toplevel_bisulfite.tar.gz) :
`http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v080_bisulfite.tar.gz`

  - Gzipped tarball of bisulfite genome for use with Bismark
  - Creation details [here](https://robertslab.github.io/sams-notebook/2018/05/08/bs-seq-mapping-olympia-oyster-bisulfite-sequencing-trimgalore-fastqc-bismark.html)
</code></pre>
<h2 id="transcriptomes_2">Transcriptomes:</h2>
<ul>
<li>
<p><a href="http://eagle.fish.washington.edu/cnidarian/Olurida_transcriptome_v3.fasta">Olurida_transcriptome_v3.fasta</a></p>
</li>
<li>
<p>MD5 = 9da3242af2be0463051ec7e1f39b2593</p>
</li>
</ul>
<h4 id="tissue-specific-transcriptomes-generated-by-katherine-silliman">Tissue-specific transcriptomes generated by Katherine Silliman</h4>
<ul>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_CA_adductor_Trinity.fasta.gz">Olurida_CA_adductor_Trinity.fasta.gz</a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_CA_ctenidia_Trinity.fasta.gz">Olurida_CA_ctenidia_Trinity.fasta.gz</a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_CA_mantle_Trinity.fasta.gz">Olurida_CA_mantle_Trinity.fasta.gz</a></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_gonad_Trinity.fasta.gz">Olurida_gonad_Trinity.fasta.gz</a></p>
</li>
</ul>
<h2 id="genome-feature-tracks_2">Genome Feature Tracks</h2>
<ul>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_genome_snap02.all.renamed.putative_function.domain_added.gff">Olurida_v081_genome_snap02.all.renamed.putative_function.domain_added.gff</a> (2.9GB) : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_genome_snap02.all.renamed.putative_function.domain_added.gff</code></p>
</li>
<li>
<p>MD5 = <code>f54512bd964f45645c34b1e8e403a2b0</code></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.CDS.gff">Olurida_v081-20190709.CDS.gff</a> : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.CDS.gff</code></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.exon.gff">Olurida_v081-20190709.exon.gff</a> : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.exon.gff</code></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.gene.gff">Olurida_v081-20190709.gene.gff</a> : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.gene.gff</code></p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.mRNA.gff">Olurida_v081-20190709.mRNA.gff</a> : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081-20190709.mRNA.gff</code></p>
</li>
<li>
<p><a href="http:/owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_TE-Cg.gff">Olurida_v081_TE-Cg.gff</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_TE-Cg.gff</code></p>
</li>
<li>
<p>MD5 = 977fd7cdb460cd0b9df5e875e1e880ea</p>
</li>
<li>
<p>Transposable Element track - more details in <a href="http://onsnetwork.org/kubu4/2018/07/03/transposable-element-mapping-olympia-oyster-genome-assembly-olurida_v081-using-repeatmasker-4-07/">Sam's Notebook</a>, including a <a href="http://owl.fish.washington.edu/Athaliana/20180702_oly_repeatmasker_Cgigas/Olurida_v081.fa.tbl">summary table</a>.</p>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_CG-motif.gff">Olurida_v081_CG-motif.gff</a> : <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Olurida_v081_CG-motif.gff</code></p>
</li>
</ul>
<hr />
<h1 id="panopea-generosa"><em>Panopea generosa</em></h1>
<h2 id="genome_3">Genome:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Pgenerosa_v074.fa">Pgenerosa_v074.fa</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Pgenerosa_v074.fa</code></p>
</li>
<li>
<p><a href="https://www.ebi.ac.uk/ena/data/view/GCA_902825435">ENA Accession: GCA_902825435</a></p>
</li>
<li>
<p>Version of 070 containing 18 largest scaffolds (<a href="https://robertslab.github.io/sams-notebook/2019/06/25/Data-Wrangling-FastA-Subsetting-of-Pgenerosa_v070.fa-Using-samtools-faidx.html">details on subsetting</a>)</p>
</li>
<li>
<p>MD5 = 32976550b9030126c07920d5f2db179c</p>
</li>
<li>
<p>BUSCO scores:</p>
<ul>
<li><code>C:71.6%[S:70.7%,D:0.9%],F:4.7%,M:23.7%,n:978</code></li>
<li><a href="https://robertslab.github.io/sams-notebook/2019/07/10/Genome-Assessment-BUSCO-Metazoa-on-Pgenerosa_v074-on-Mox.html">Notebook entry</a></li>
</ul>
</li>
<li>
<p>FastA index (<code>samtools faidx</code>) - <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Pgenerosa_v074.fa.fai</code></p>
</li>
</ul>
<h2 id="bisulfite-genome_1">Bisulfite Genome</h2>
<h2 id="genome-feature-tracks_3">Genome Feature Tracks</h2>
<h4 id="panopea-generosa-vv074a3">Panopea-generosa-vv0.74.a3</h4>
<ul>
<li>
<p><em>These originate from <a href="https://robertslab.github.io/sams-notebook/2019/07/10/Genome-Annotation-Pgenerosa_v074-Using-GenSAS.html">GenSAS annotation on 20190710</a></em></p>
</li>
<li>
<p>BUSCO:</p>
</li>
<li>
<p>68.4% complete BUSCOs</p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.gene.gff3">Panopea-generosa-vv0.74.a3.gene.gff3</a> :
  <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.gene.gff3</code></p>
<ul>
<li><a href="https://d.pr/n/RHThn4">code</a></li>
</ul>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.exon.gff3">Panopea-generosa-vv0.74.a3.exon.gff3</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.exon.gff3</code></p>
</li>
<li>
<p><a href="https://d.pr/n/RHThn4">code</a></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.intragenic.bed">Panopea-generosa-vv0.74.a3.intragenic.bed</a> : <code>https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.intragenic.bed</code></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.intron.bed">Panopea-generosa-vv0.74.a3.intron.bed</a> : <code>https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.intron.bed</code></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.rm.gff3">Panopea-generosa-vv0.74.a3.rm.gff3</a> : <code>https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.rm.gff3</code></p>
</li>
<li>
<p>index: <code>https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Panopea-generosa-vv0.74.a3.rm.gff3.idx</code></p>
</li>
<li>
<p><a href="https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Pgenerosa_v074.CpG.gff">Pgenerosa_v074.CpG.gff</a> : <code>https://gannet.fish.washington.edu/seashell/wd/091319/tracks/Pgenerosa_v074.CpG.gff</code></p>
</li>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.CDS.gff3">Panopea-generosa-vv0.74.a3.CDS.gff3</a> :
  <code>https://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.CDS.gff3</code></p>
<ul>
<li><a href="https://d.pr/n/RHThn4">code</a></li>
</ul>
</li>
<li>
<p><a href="https://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.mRNA.gff3">Panopea-generosa-vv0.74.a3.mRNA.gff3</a> :
<code>https://owl.fish.washington.edu/halfshell/genomic-databank/Panopea-generosa-vv0.74.a3.mRNA.gff3</code></p>
</li>
<li>
<p><a href="https://d.pr/n/RHThn4">code</a></p>
</li>
</ul>
<hr />
<h2 id="transcriptome">Transcriptome:</h2>
<ul>
<li>
<p><a href="http://owl.fish.washington.edu/halfshell/genomic-databank/Pgenerosa_transcriptome_v5.fasta">Pgenerosa_transcriptome_v5.fasta</a> : <code>http://owl.fish.washington.edu/halfshell/genomic-databank/Pgenerosa_transcriptome_v5.fasta</code></p>
</li>
<li>
<p>MD5 = 5a21424ecbc88c3b01daefe56bed79da</p>
</li>
</ul>
<p>Transcriptome generated from various libaries - details <a href="https://robertslab.github.io/sams-notebook/2018/09/04/transcriptome-assembly-geoduck-rnaseq-data.html">here</a></p>
<hr />
<h1 id="qpx"><em>QPX</em></h1>
<h2 id="genome_4">Genome:</h2>
<ul>
<li><a href="http://eagle.fish.washington.edu/QPX_genome/QPX_v017.fasta">QPX_v017.fasta</a> : <code>http://eagle.fish.washington.edu/QPX_genome/QPX_v017.fasta</code></li>
</ul>
<p>CLC v5.1 Mismatch cost = 2; Perform scaffolding = Yes; Mapping mode = Map reads back to contigs (slow); Deletion cost = 3; Similarity fraction = 0.9; Length fraction = 0.8; Insertion cost = 3; Update contigs = Yes; Automatic word size = Yes; Minimum contig length = 10000; Automatic bubble size = Yes; input: filtered_QPX_DNA_GTGAAA_L001_R1 trimmed.</p>
<ul>
<li><a href="https://ndownloader.figshare.com/files/3085550">QPX_v017.fasta</a> : <code>https://ndownloader.figshare.com/files/3085550</code></li>
</ul>
<p>CLC v5.1 Mismatch cost = 2; Perform scaffolding = Yes; Mapping mode = Map reads back to contigs (slow); Deletion cost = 3; Similarity fraction = 0.9; Length fraction = 0.8; Insertion cost = 3; Update contigs = Yes; Automatic word size = Yes; Minimum contig length = 10000; Automatic bubble size = Yes; input: filtered_QPX_DNA_GTGAAA_L001_R1 trimmed.</p>
<ul>
<li><a href="https://doi.org/10.1371/journal.pone.0074196.s001">QPX_v015.fasta</a> : <code>https://doi.org/10.1371/journal.pone.0074196.s001</code></li>
</ul>
<blockquote>
<p>De novo assembly was performed with Genomics Workbench v. 5.0 (CLC Bio, Germany) on quality trimmed sequences with the following parameters: mismatch cost = 2, deletion cost = 3, similarity fraction = 0.9, insertion cost = 3, length fraction = 0.8 and minimum contig size of 100 bp for genomic data and 200 bp for transcriptomic data. In order to remove ribosomal RNA sequences from the transcriptome data, consensus sequences were compared to the NCBI nt database using the BLASTn algorithm [59]. Sequences with significant matches (9) were removed and not considered in subsequent analyses.</p>
</blockquote>
<p>Manuscript: https://doi.org/10.1371/journal.pone.0074196</p>
<h2 id="transcriptome_1">Transcriptome:</h2>
<ul>
<li><a href="http://eagle.fish.washington.edu/cnidarian/QPX_transcriptome_v1_clean.fasta">QPX_transcriptome_v1_clean.fasta</a></li>
</ul>
<p>QPX_Transcriptome v2.1</p>
<p>Subset of version 1 (v1) that only includes sequences with e-value &lt; 1E-20. Based on Swiss-Prot blastx output, all sequences are oriented 5' - 3'. nucleotides between stop codons; minimum size 200.</p>
<hr />
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid" aria-label="Footer">
        
          <a href="../External-Communication-%26-Funding/" class="md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
            </div>
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                External Communication & Funding
              </div>
            </div>
          </a>
        
        
          <a href="../Graduate-Research-Plan/" class="md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Graduate Research Plan
              </div>
            </div>
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
          Material for MkDocs
        </a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/vendor.93c04032.min.js"></script>
      <script src="../assets/javascripts/bundle.83e5331e.min.js"></script><script id="__lang" type="application/json">{"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}</script>
      
      <script>
        app = initialize({
          base: "..",
          features: [],
          search: Object.assign({
            worker: "../assets/javascripts/worker/search.8c7e0a7e.min.js"
          }, typeof search !== "undefined" && search)
        })
      </script>
      
    
  </body>
</html>