TOC   
* [hummingbird](https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-hummingbird)    
* [roadrunner](https://github.com/RobertsLab/resources/wiki/Lab-Software/#computer-roadrunner)
* [ostrich](https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-ostrich)
* [emu](https://github.com/RobertsLab/resources/wiki/Lab-Software/#computer-emu)
* [woodpecker](https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-woodpecker)
* [Mox](https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-mox-hyak)
* [swan](https://github.com/RobertsLab/resources/wiki/Lab-Software#computer-swan)

---

### Computer: hummingbird  
#### OS: OS X (10.9.5)  

FastQC (0.11.2)

     path = /usr/local/bioinformatics/FastQC/

Trimmomatic (0.30)

     path = /usr/local/bioinformatics/Trimmomatic-0.30/

Trinotate (Release 2014-07-08)

     path = /usr/local/bioinformatics/Trinotate_r20140708/

bedtools (2.22.1)

     path = /usr/local/bioinformatics/bedtools-2.22.1/

bismark (0.14.2)

     path = /usr/local/bioinformatics/bismark_v0.14.2/

bowtie (1.0.0)

     path = /usr/local/bioinformatics/bowtie-1.0.0/

bowtie2 (2.1.0)

     path = /usr/local/bioinformatics/bowtie2-2.1.0/

cufflinks (2.2.1.OSX_x86_64)

     path = /usr/local/bioinformatics/cufflinks-2.2.1.OSX_x86_64/

fastx_toolkit (0.0.13.2)

     path = /usr/local/bioinformatics/fastx_toolkit-0.0.13.2/

hmmer (3.1b1)

     path = /usr/local/bioinformatics/hmmer-3.1b1/

libgtextutils (0.6.1)

     path = /usr/local/bioinformatics/libgtextutils-0.6.1/

ncbi-blast (2.2.29+)

     path = /usr/local/bioinformatics/ncbi-blast-2.2.29+/

ngsplot (2.08)

     path = /usr/local/bioinformatics/ngsplot-2.08/

rsem (1.2.10)

     path = /usr/local/bioinformatics/rsem-1.2.10/

signalp (4.1)

     path = /usr/local/bioinformatics/signalp-4.1/

stacks (1.13)

     path = /usr/local/bioinformatics/stacks-1.13/

tophat (2.0.13.OSX_x86_64)

     path = /usr/local/bioinformatics/tophat-2.0.13.OSX_x86_64/

trinity (Release 2013-11-10)

     path = /usr/local/bioinformatics/trinityrnaseq_r20131110/

ipython (3.0.0)

Geneious (5.3.6)

----

### Computer: ostrich  
#### OS: OS X (10.11.6)  

Anaconda3 (with R kernel and rpy2 for jupyter)

     path = /Users/Shared/anaconda3

FastQC (0.11.5)

     path = /Users/Shared/bioinformatics/FastQC/

Trimmomatic (0.30)

     path = /Users/Shared/bioinformatics/Trimmomatic-0.36/

Trinotate (3.0.1)

     path = /Users/Shared/bioinformatics/Trinotate-3.0.1/

bedtools (2.26.0)

     path = /Users/Shared/bioinformatics/bedtools2/

bismark (0.16.3)

     path = /Users/Shared/bioinformatics/bismark_v0.15.0/
bowtie (1.1.2)

     path = /Users/Shared/bioinformatics/bowtie1-1.1.2

bowtie2 (2.2.9)

     path = /Users/Shared/bioinformatics/bowtie2-2.2.9

celera assembler (8.3rc2)

    path = /Users/Shared/bioinformatics/wgs-8.3rc2

cufflinks (2.2.1.OSX_x86_64)

     path = /Users/Shared/bioinformatics/cufflinks-2.2.1.OSX_x86_64

hmmer (3.1b2)

     path = /Users/Shared/bioinformatics/hmmer-3.1b2-macosx-intel

ncbi-blast (2.5.0)

     path = /Users/Shared/bioinformatics/ncbi-blast-2.5.0+

ngsplot (2.08)

     path = /Users/Shared/bioinformatics/ngsplot

rsem (1.3.0)

     path = /Users/Shared/bioinformatics/RSEM-1.3.0

stacks (1.44)

     path = /Users/Shared/bioinformatics/stacks-1.44

tophat (2.1.1)

     path = /Users/Shared/bioinformatics/tophat-2.1.1.OSX_x86_64

trinity (2.3.2)

     path = /Users/Shared/bioinformatics/trinityrnaseq-Trinity-V2.3.2

----
### Computer: emu
#### OS: Ubuntu 16.04
#### CPU(s): 16
#### Memory: 48GB
#### HDD Capacity: 1.8TB

```
$/home/shared
20200207_macau                BUSCO_v1.22                     include                         MultiQC                RepeatModeler-open-1.0.11   SparseAssembler
abacus                        comet                           isb-2.6.0+-changes-vers2.patch  ncbi-blast-2.6.0+-src  RepeatScout-1.0.5           sra-toolkit
bedtools-2.28.0               doc                             jellyfish-2.2.6                 nseg                   rmblast_2.6.0               src
BerkleyDB                     EMBOSS-6.6.0                    kallisto                        PBSuite_15.8.24        RNAMMER                     tmhmm-2.0c
bfg-1.13.0.jar                etc                             last-658                        pecan                  salmon-0.11.3-linux_x86_64  TransDecoder-3.0.1
bin                           fastqc_0.11.7                   lib                             pitchfork              salmon-1.2.1_linux_x86_64   TrimGalore-0.4.5
Bismark-0.19.1                gsl                             macau                           ProtDigSim             samtools-1.8                Trimmomatic-0.36
bissnp                        hmmer-2.3                       methpipe                        proteowiz              seqkit.v0.9.3               trinityrnaseq-Trinity-v2.4.0
BisSNP                        hmmer-3.1b2-linux-intel-x86_64  MethylExtract_1.9.1             pyfaidx-0.5.5.2        SGE6.2u5p2                  trinityrnaseq-Trinity-v2.8.4
blastdb                       IGV_Linux_2.6.1                 miniconda2                      ReAdW                  share                       Trinotate-3.0.2
bowtie2-2.3.4.1-linux-x86_64  igvtools                        miniconda3                      RECON-1.08             Sibelia-3.0.7-Linux         var
bsmap                         igv-webapp                      MSFileReader                    RepeatMasker-4.0.7     signalp-4.1
```

```
/usr/bin
R (Microsoft Open R 3.5.1)
```

```
/home/shared/miniconda3/bin
jupyter (with R Kernel)
```

----
### Computer: roadrunner  
#### OS: Ubuntu 16.04  
#### CPU(s): 16
#### Memory: 48GB
#### HDD Capacity: 1.8TB

```
$/home/shared
alpaca           Bismark-v0.17.0               bwa-0.7.17      igv-webapp              macau                  ncbi-blast-2.7.1+-src  RepeatMasker-4.0.7         SOAPdenovo2-r241
anaconda3        BisSNP                        ectools-0.1     jellyfish-2.2.6         meraculous-2.2.5       parallel-20180422      rmblast_2.6.0              TrimGalore-0.4.5
bbmap_38.44      bowtie2-2.3.3.1-linux-x86_64  EMBOSS-6.6.0    kallisto_linux-v0.43.1  Meraculous-v2.2.5      PBSuite_15.8.24        salmon-1.2.1_linux_x86_64  wgs-assembler
bedtools-2.28.0  bowtie2-2.3.4.1-linux-x86_64  fastqc_v0.11.7  kmergenie-1.7044        MultiQC                pecan                  samtools-1.6
bin              BSMap                         IGVTools        Log-Log4perl-1.49       ncbi-blast-2.6.0+-src  pyfaidx-0.5.3.1        samtools-1.8
```

```
/usr/bin
cutadapt
R (Microsoft Open R 3.5.1)
```
```
/home/shared/anaconda3/bin
jupyter (with R Kernel)
```

##### [JBrowse Genome Viewer v1.15.4](https://jbrowse.org/)

http://roadrunner.fish.washington.edu/jbrowse/

---
### Computer: Mox (Hyak)
#### OS: Red Hat Enterprise 7.2 ([login nodes](https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#login-node))
#### OS: CentOS Linux 7.2.1511 ([build](https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#build-node),  [interactive nodes](https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#interactive-node), & [execute nodes](https://github.com/RobertsLab/hyak_mox/wiki/Node-Types#execute-node))

```
$/gscratch/srlab/programs
2bRAD_GATK		      busco-v3		git-sym			mdust					 perl			     samtools-1.4		       supernova-2.0.0
argparse-1.4.0		      bwa-0.7.15	graphviz-2.40.1		megahit_v1.1.4_LINUX_CPUONLY_x86_64-bin  picard_2.18.4		     samtools-1.9		       tmhmm-2.0c
Augustus-3.3.2		      bwa-0.7.17	hisat2-2.1.0		MetaGeneMark_linux_64_3.38		 picard-2.9.1		     scripts			       transdecoder
bamtools-2.5.1		      canu		hmmer-2.3		Metassembler				 pigz-2.4		     seqtk-1.3			       TransDecoder-v5.3.0
bbmap_38.34		      celera		hmmer-3.1b2		minimap					 pilon			     setuptools-36.0.1		       TransDecoder-v5.5.0
bcftools-1.9		      cmake-3.12.1	hmmer-3.2.1		mummer					 pitchfork		     signalp-4.1		       TrimGalore-0.4.5
bcl2fastq-v2.20		      cutadapt-1.16	htslib-1.9		MUMmer3.23				 platanus_1.2.4		     smrtanalysis_2.3.0.140936.run     Trimmomatic-0.36
bedtools-2.27.1		      decorator-4.0.11	interproscan-5.31-70.0	ncbi-blast-2.6.0+			 pyfaidx-0.5.5.2	     snap			       Trinity-v2.8.3
bin			      FastaIndex	jellyfish-2.2.10	ncbi-blast-2.8.1+			 pyScaf			     SOAPdenovo2		       Trinotate-v3.1.1
Bismark-0.19.0		      FastQC-0.11.5	jsoncpp-1.8.4		ncbi-blast-2.8.1+_orginal		 quast-4.5		     SOAPec_bin_v2.03		       wtdbg-2.1_x64-linux
Bismark-0.21.0		      gapcloser-1.12	kmergenie-1.7048	networkx-1.11				 racon			     SPAdes-3.13.0-Linux
Bismark-0.21.0_dev	      GARM_v0.7.5	last-852		networkx2				 redundans		     SparseAssembler
bowtie2-2.1.0		      GATK		likelybin-0.1.0		parallel-20180822			 resources		     sqlite
bowtie2-2.3.4.1-linux-x86_64  gcc		maker-2.31.10		PBSuite					 RNAMMER-1.2		     SSPACE-LongRead_v1-1
bsmap-2.89		      get-pip.py	MaSuRCA-3.2.3		pecan					 salmon-0.11.2-linux_x86_64  SSPACE-STANDARD-3.0_linux-x86_64

```


---
### Computer: woodpecker  
#### OS: Windows 7 Enterprise (64-bit)

Microsoft Office 2016 Pro Plus

          path = C:\Program Files\Microsoft Office

Protein Digest Simulator v2.2.6138.19320

          path = C:\Program Files (x86)\ProteinDigestionSimulator

Proteowizard v3.0.10577

          path = C:\Program Files\ProteoWizard\ProteoWizard 3.0.10577

Skyline v3.6

          path =

---

### Computer: swan
#### OS: Windows 7 Enterprise (64-bit)
Microsoft Office Professional 2016 64-bit
- Access
- Excel
- Power Point
- Word

R v3.4.2

R Studio 1.1383

Skyline v3.7.0.11317

Skyline Daily v3.7.1.11446

Virtual Clonedrive v5.5.0.0

ProteoWizard 3.0.11516 64-bit
- MSConvert
- SeeMS

LabX v. 8.0.0 Build 3444

Protein Digest Simulator v2.2.6471; September 19, 2017
